src/RestClient/endpoints/V3/Themes/ThemeConfigurations.ts

```diff
@@ -0,0 +1,37 @@
+import { AxiosInstance } from 'axios';
+
+import { themesPath } from './Themes';
+import type { ThemeConfiguration } from './types';
+
+class ThemeConfigurations {
+  private client: AxiosInstance;
+
+  constructor(client: AxiosInstance) {
+    this.client = client;
+  }
+
+  /**
+   * Returns a list of theme configurations
+   *
+   * @param uuid A valid Theme UUID
+   * @param params Query parameters used to filter response
+   * @returns Promise resolving to a list of theme configurations
+   */
+  list(uuid: string, params?: ThemeConfiguration['ListFilters']): ThemeConfiguration['ListResponse'] {
```

Comment:
🍹 This is an interesting case for `params`, as our documentation states that "At least one filter must be provided."

There are some Typescript tricks that we may want to employ here. See: https://stackoverflow.com/a/49725198/8211101